templates/repo/settings/options.tmpl

```diff
@@ -209,13 +211,19 @@
 												{{.i18n.Tr "repo.need_auth"}}
 											</summary>
 											<div class="p-2">
-												<div class="inline field {{if .Err_PushMirrorAuth}}error{{end}}">
-													<label for="push_mirror_username">{{.i18n.Tr "username"}}</label>
-													<input id="push_mirror_username" name="push_mirror_username" value="{{.push_mirror_username}}">
-												</div>
-												<div class="inline field {{if .Err_PushMirrorAuth}}error{{end}}">
-													<label for="push_mirror_password">{{.i18n.Tr "password"}}</label>
-													<input id="push_mirror_password" name="push_mirror_password" type="password" value="{{.push_mirror_password}}" autocomplete="off">
+												<div>
+													<div class="inline field {{if .Err_PushMirrorAuth}}error{{end}}">
+														<label for="push_mirror_username">{{.i18n.Tr "username"}}</label>
+														<input id="push_mirror_username" name="push_mirror_username" value="{{.push_mirror_username}}">
+													</div>
+													<div class="inline field {{if .Err_PushMirrorAuth}}error{{end}}">
+														<label for="push_mirror_password">{{.i18n.Tr "password"}}</label>
+														<input id="push_mirror_password" name="push_mirror_password" type="password" value="{{.push_mirror_password}}" autocomplete="off">
+													</div>
+													<div class="inline field {{if .Err_PushMirrorUsePublicKey}}error{{end}}">
+														<label for="push_mirror_use_public_key">{{.i18n.Tr "repo.mirror_use_public_key"}}</label>
+														<input name="push_mirror_use_public_key" type="checkbox" {{.push_mirror_use_public_key}}>
```

Comment:
`type="checkbox" {{.push_mirror_use_public_key}}>` what is this ....... will the checkbox be checked if there is a public key?